src/shared/components/query/QueryStore.ts

```diff
@@ -2009,7 +2010,7 @@ export class QueryStore {
             : (profileIds.filter(_.identity) as string[]);
         this.zScoreThreshold = params.Z_SCORE_THRESHOLD || '2.0';
         this.rppaScoreThreshold = params.RPPA_SCORE_THRESHOLD || '2.0';
-        this.dataTypePriorityCode = params.data_priority || '0';
+        this.dataTypePriorityCode = params.data_priority || params.profileFilter ||  '0';
```

Comment:
can this parsing be brought into urlwrapper since its handling the aliasing?

Reply:
truth be known (is that a phrase? no, but i still like it), we could do away with params.data_priority here because it's an instance of urlWrapper that is being passed.  But then we would have to do some refactoring and i want to just get this out.  we can follow up with refactoring.  won't be too hard